src/common/config/chartTypes/bar.js

```diff
@@ -1,7 +1,85 @@
+import colorPalettes from '../colorPalettes';
+
+const BackgroundColor = getComputedStyle(
```

Comment:
This for example is defined but never used.

Reply:
Yeah been doing a lot of experimentation. Will do a cleanup pass before merge.